paper/worcs.bib

```diff
@@ -418,6 +418,14 @@ @article{peikertReproducibleDataAnalysis2019
   file = {C\:\\Users\\lissa102\\Zotero\\storage\\AS9JL74K\\Peikert_Brandmaier_2019_A Reproducible Data Analysis Workflow with R Markdown, Git, Make, and Docker.pdf;C\:\\Users\\lissa102\\Zotero\\storage\\2NT8MMMG\\8xzqy.html}
 }
 
+@Manual{repro,
+  title = {repro: Automated Setup of Reproducible Workflows and their Dependencies},
+  author = {Peikert, Aaron and Brandmaier, Andreas Markus and {van Lissa}, {Caper J.}},
```

Comment:
Caper is a nice pizza topping, but I prefer to go by Caspar

Reply:
Oh I am sorry... That is what happens if I do something by hand instead of automation.